test/functional/normal/fold_spec.lua

```diff
@@ -44,4 +48,113 @@ describe('Folds', function()
     8
     9]])
   end)
+  it('updates correctly on :read', function()
+    -- luacheck: ignore 621
+    helpers.write_file('./to_read.txt', [[
+    a
+
+
+    	a]])
+    -- luacheck: ignore 621
```

Comment:
this shouldn't be necessary since the case on line 52 ignores it for the entire current scope.